README.md

```diff
@@ -262,12 +262,12 @@ for parsing a value out of the JSON:
 
 The usage of these operators is the same regardless:
 
-- `json <| "key"` is analogous to `json["key"]`
-- `json <| ["key", "nested"]` is analogous to `json["key"]["nested"]`
+- `json <| "key"` will decode the value for the key "key"
+- `json <| ["key", "nested"]` will decode the value at '{ "key": { "nested"'
 
-Both operators will attempt to parse the value from the JSON and will also
-attempt to cast the value to the expected type. If it can't find a value, or
-if that value is of the wrong type, the function will return `.None`.
+Both operators will attempt to decode the value from the JSON into the
```

Comment:
Reiterating what Joe mentioned, I think I like the Aeson term "retrieve." 
So you PARSE the json and in DECODE you RETRIEVE values.
Does that sound logical?